helm-charts/metal3-deploy/charts/baremetal-operator/templates/configmap-ironic.yaml

```diff
@@ -13,6 +13,7 @@ data:
   PROVISIONING_IP: {{ .Values.global.provisioningIP }}
   IRONIC_FAST_TRACK: "true"
   RESTART_CONTAINER_CERTIFICATE_UPDATED: "false"
+  IRONIC_INSECURE: "true"
```

Comment:
why is this set to "true"?

Reply:
Since tls is not implemented in baremetalhost, we decided to use this flag and document that work needs to be done on baremetalhost side. The baremetalhost won't be able to talk to ironic if this flag is not set.

Reply:
I see. Thanks.